src/ontology/uberon-edit.obo

```diff
@@ -224517,6 +224517,20 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8850000
+name: lumen of alveolus
+def: "An anatomical cavity that is part of the alveolus of the lung" [PMID:30390118]
+synonym: "Alveolar lumen" EXACT []
+synonym: "Cavity of pulmonary alveolus" EXACT [FMA:62779]
+synonym: "Lumen of pulmonary alveolus" EXACT [FMA:62779]
+synonym: "Pulmonary alveolar cavity" EXACT [FMA:62779]
+intersection_of: UBERON:0000464 ! anatomical space
+intersection_of: luminal_space_of UBERON:0002299 ! alveolus of lung
```

Comment:
I think it would be good to have cohesion between the textual definition and the logical definition. `anatomical cavity` is a subclass of `anatomical space`, and is defined as follows:
> Anatomical space which contains portions of one or more body substances and is bounded by the internal surface of one maximally connected anatomical structure. Examples: cranial cavity, pharyngeal recess space, nasal cavity, tooth socket, cavity of serous sac, lumen of stomach, lumen of artery, fornix of vagina.
From what I have seen looking for this, there are some lumens that are subclass of anatomical cavity and others that are directly subclass of anatomical space. I would favour `anatomical cavity, what match with you textual definition.